vscode/src/common.ts

```diff
@@ -85,6 +85,7 @@ export const SUPPORTED_LANGUAGE_IDS = ["ruby", "erb"];
 export const FEATURE_FLAGS = {
   tapiocaAddon: 0.3,
   launcher: 0.05,
+  newTestExperience: -1,
```

Comment:
Can we give a more descriptive name here, maybe `lazyTestDiscovery`? "new" becomes a less useful distinction as time goes on.

Reply:
Changed the name to `fullDiscovery`. The intention is to have everyone migrate to the new experience, so the flag is temporary.